.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I usually add `.vscode` to .`gitignore` as it often contains user-specific settings. However, if this has been a working approach in your previous projects, I'm not opposed to it 🙂

Reply:
I also intuitively thought of adding it, but then again since this project hopefully will be worked on by a lot of people 😄  Adding automatic formatting on save might be useful? As long as we all use the same editor, and not get specific config folders for like 10 different editors/IDEs here...